core/google/api/core/retry.py

```diff
@@ -150,6 +150,9 @@ def retry_target(target, predicate, sleep_generator, deadline):
         sleep_generator (Iterator[float]): An infinite iterator that determines
             how long to sleep between retries.
         deadline (float): How long to keep retrying the target.
+        on_error(Callable): A function to call while processing a retryable
```

Comment:
nit: Needs a space between `on_error` and `(Callable)`.

Reply:
e339f94